agent/eni/networkutils/utils_noop.go

```diff
@@ -0,0 +1,17 @@
+// +build !linux
```

Comment:
These files are not just for mock. They help with the overall build support. 
I'll follow up with an update